pkg/staticnetworkconfig/generator.go

```diff
@@ -101,6 +102,9 @@ func (s *StaticNetworkConfigGenerator) createNMConnectionFiles(nmstateOutput, ho
 		return nil, err
 	}
 	filesList := []StaticNetworkConfigData{}
+	if _, found := hostNMConnections["NetworkManager"]; !found {
+		return filesList, errors.Errorf("host NetworkManager is empty")
```

Comment:
in case of an error, no need to return the filesList - nil will be enough.
then the `if` can be moved to line 104, before declaring on `filesList`

also, the message can be more descriptive to this case, such as: "nmstate generated an empty network-manager config file content"

Reply:
Done